test/tools/cluster.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Makes this a bit more portable